src/routers/nomination/index.ts

```diff
@@ -13,9 +13,14 @@ router.post(
   '/',
   getValidations(LocalRoutes.CREATE_NOMINATION),
   validate,
-  auth,
+  auth(),
   async (req: AuthenticatedRequest, res: Response, next: NextFunction) => {
     try {
+      const existingNomination = await NominationModel.findOne({
+        position: req.body.position,
+        candidate: req.body.candidate,
+      });
+      if (existingNomination) throw new Error('Nomination already exists');
```

Comment:
If we use `Model.count` here we avoid reading the doc from the DB.
https://stackoverflow.com/questions/27482806/check-if-id-exists-in-a-collection-with-mongoose

```suggestion
      const existingNominationCount = await NominationModel.count({
        position: req.body.position,
        candidate: req.body.candidate,
      });
      if (existingNominationCount > 0) throw new Error('Nomination already exists');
```

Reply:
Actually I think we can even do `Model.exists`

Reply:
Thanks for the recommendation! Updated here: c03449a194f061dee5ba595ae3e03f3c2fd66bf4